.github/workflows/pr-chart-lint-and-test.yml

```diff
@@ -50,10 +50,6 @@ jobs:
             kindest_image: kindest/node:v1.24.15@sha256:7db4f8bea3e14b82d12e044e25e34bd53754b7f2b0e9d56df21774e6f66a70ab
           - name: k8s-1.23
             kindest_image: kindest/node:v1.23.17@sha256:59c989ff8a517a93127d4a536e7014d28e235fb3529d9fba91b3951d461edfdb
-          - name: k8s-1.22
-            kindest_image: kindest/node:v1.22.17@sha256:f5b2e5698c6c9d6d0adc419c0deae21a425c07d81bbf3b6a6834042f25d4fba2
-          - name: k8s-1.21
```

Comment:
these lines https://github.com/MediaMarktSaturn/helm-charts/blob/main/.github/workflows/pr-chart-lint-and-test.yml#L115-L116
need to be removed as well

Reply:
thx @beiertu-mms